include/beast/http/impl/write.ipp

```diff
@@ -170,13 +148,13 @@ class write_op
         }
     };
 
-    class writef0
+    class writef0_lambda
```

Comment:
I don't see where the equivalent lambda would use any C++-14 features. Why the work-around class? (same comment for `writef_lambda`)